feat: added a config option for the copy text

nick22985 opened the pull request:
- Added the ability to change the [copy] -> [{value}] to whatever value the user wishes

nick22985 committed:
feat: added a config option for the copy text

Fesaa commented:
Heya! Thanks for the contribution! ❤️

Is there a reason you're not giving users the ability to modify the brackets and colour? If I'm going to add custom things to do button I think I'd just have everything be custom. 

Let me know if you're willing to add this, or if you have a specific reason not to. If you don't have the time I'll do it after so no worries ☺️

Fesaa commented on core/src/main/java/org/chatutilities/core/config/MainConfig.java:
```diff
@@ -19,6 +20,9 @@ public class MainConfig extends AddonConfig {
   private final ConfigProperty<Boolean> copy = new ConfigProperty<>(false);
 
+  @TextFieldSetting
+  private final ConfigProperty<String> copyButtonText = new ConfigProperty<>("Copy");
```
Could you change the `"Copy"` to `I18n.translate(...)` such that the default value aligns with the language the user has selected. 
I know there currently is only English, but just future proofing.

nick22985 replied:
Hey,
As I am relatively new to laby mod addons. I am not really sure how to do this or the best way this should be done.

Fesaa replied:
Heya! No worries, appears to not be possible anyway. The config is initialized before the localization is 🥲

nick22985 committed:
feat: Made copy brackets configurable. Added configuration to copy tooltips

nick22985 commented:
> Heya! Thanks for the contribution! ❤️
> 
> Is there a reason you're not giving users the ability to modify the brackets and colour? If I'm going to add custom things to do button I think I'd just have everything be custom.
> 
> Let me know if you're willing to add this, or if you have a specific reason not to. If you don't have the time I'll do it after so no worries ☺️

Hey,
I have removed the brackets from the message and made it so that it is in the format. Also added support for colour codes in the format message. Unsure if this is the way you want to go about this or not.

Fesaa committed:
Unused code, reset format, plugin version

Fesaa commented:
Well, workflow not set up correctly for pull requests…

Anyway, thanks for the contribution! I'll merge it now, and submit an update to LabyMod once (/if) it builds. ❤️
